addon-test-support/@ember/test-helpers/dom/blur.ts

```diff
@@ -24,14 +29,17 @@ export function __blur__(element: HTMLElement | Element | Document | SVGElement)
 
   // makes `document.activeElement` be `body`.
   // If the browser is focused, it also fires a blur event
```

Comment:
I'd like to keep this comment with the `element.blur()` call.

Reply:
Done.